pkg/server/asset/cloud.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2015-present Oursky Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package asset
+
+import (
+	"errors"
+	"io"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/franela/goreq"
+)
+
+const (
+	cloudAssetSignerTokenRefreshInterval = 30 * time.Minute
+	cloudAssetSignerTokenExpiryInterval  = 2 * time.Hour
+)
+
+// CloudStore models the skygear cloud asset store
+type CloudStore struct {
```

Comment:
Since this struct only implements other interface, you might just set this struct private.